### PR TITLE
Item list improvements

### PR DESCRIPTION
**Pull request by F-Fischer**
News interface added, Item List component added and Item component changes to show title, domain, points, user and time ago attributes

### CODE REVIEW

**Review by lakardion — APPROVED** [review]
lgtm.
I'd like to a see the preview app but  I just set it up so it might not show until next commit
https://react-ors3tmkpu-lakardion.vercel.app/

File: src/components/ItemList.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { News } from "../models/News";
```
Comment (lakardion) [inline src/components/ItemList.tsx]: I'd rather have file names to be kebab (item-list). This is mainly due to issues with mac filesystem being case insensitive. I always avoid casing files in projects unless the project already has pascal-case set up

Comment (F-Fischer) [reply src/components/ItemList.tsx]: I changed the file names, and added some new styles to the item component if you wanna check.

**Comment by lakardion** [comment]
```
src/App.tsx(3,22): error TS2307: Cannot find module './models/news' or its corresponding type declarations.
--
09:35:10.014 | src/App.tsx(23,19): error TS2322: Type 'unknown' is not assignable to type 'News[] \| undefined'.
09:35:10.014 | src/components/item-list.tsx(1,22): error TS2307: Cannot find module '../models/news' or its corresponding type declarations.
09:35:10.015 | src/components/item.tsx(2,22): error TS2307: Cannot find module '../models/news' or its corresponding type declarations.
09:35:10.015 | src/services/hn.ts(1,22): error TS2307: Cannot find module '../models/news' or its corresponding type declarations.
09:35:10.048 | Error: Command "npm run build" exited with 2

```

**Pull request merged**